libs/platform/experience/src/routes.ts

```diff
@@ -18,31 +18,30 @@ export const defaultExperienceRoutes: RouteConfig[] = [
         .resolveToken('USER.AUTHENTICATED')
         .pipe(
           take(1),
-          map((state) => {
-            if (state) {
-              resolve(RouterService).navigate('/');
-            }
-
-            return !state;
-          })
+          map((state) => (state ? '/' : !state))
         ),
   },
-  {
-    path: '/product/:sku',
-    type: RouteType.Product,
-  },
-  {
-    path: '/category/:id',
-    type: RouteType.Category,
-  },
   {
     path: '/:page',
     type: RouteType.Page,
   },
   {
     path: '/*',
+    type: RouteType.NotFound,
     render: (): TemplateResult =>
       html`<oryx-heading><h1>Error 404</h1></oryx-heading>
         <p>Page not found</p>`,
   },
 ];
+
+export const defaultExperienceRoutes: RouteConfig[] = [
```

Comment:
change order of elements in const array could also be considered a breaking change